contracts/cw20-merkle-airdrop/src/state.rs

```diff
@@ -0,0 +1,23 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use cosmwasm_std::Addr;
+use cw_storage_plus::{Item, Map, U8Key};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct Config {
+    pub owner: Addr,
+    pub cw20_token_address: Addr,
+}
+
+pub const CONFIG_KEY: &str = "config";
+pub const CONFIG: Item<Config> = Item::new(CONFIG_KEY);
+
+pub const STAGE_KEY: &str = "stage";
```

Comment:
2 points:

1) we don't need to store the key separately with storage-plus, I would just combine these into one line. (same with others). `pub const STAGE: Item<u8> = Item::new("stage);` works well
2) I assume this is *latest* stage. Can you change the name or add rustdoc to clarify that

Reply:
> we don't need to store the key separately with storage-plus, I would just combine these into one line. (same with others). pub const STAGE: Item<u8> = Item::new("stage); works well

Purpose of storing key separately is for making better raw query coding, when raw querying importing contract can used this variable to build a key. Thought this could be a good practice

Reply:
I avoid supporting raw queries, except in a few special cases.
cw4 interface is one and has a very well define raw query support for ONE use case - is_member.

But if you prefer them, your style is not wrong, just a bit more verbose. And yes, does allow raw queries.

Reply:
> I avoid supporting raw queries, except in a few special cases.
Hmm why avoid?
I see many times contracts calling each others config items. IMO advocating item raw query usage could reduce gas cost chain wide.